eventmesh-catalog-go/go.mod

```diff
@@ -29,10 +29,72 @@ require (
 	gorm.io/gorm v1.23.8
 )
 
+require (
+	github.com/KyleBanks/depth v1.2.1 // indirect
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
```

Comment:
@fengshunli hello，why add these indirect dependencies?

Reply:
This is an indirect dependency that pulls in other modules to generate

Reply:
yeah, It means why these indirect dependencies are introduced. I don’t think the current go mod introduces new dependencies. Because after adding these dependencies, you need to synchronously add licenses files in the project

Reply:
Indirect dependencies are automatically listed by the go1.18 version update mod, and do not need to be processed

Reply:
get